### PR TITLE
Make OrdersWdget moar responsive

### PR DESCRIPTION
**Pull request by W3stside**
1. Adds mobile card to Orders: Active/Liq/Closed
2. Foldable cards (start closed for space)
3. better css

**sorry for all the noise, CSS can be super annoying**

To test:
1. head to `/orders`
2. make browser mobile sized
3. mess with cards open/close
4. let me know which rows to hide/keep in open/closed mode

### CODE REVIEW

**Comment by W3stside** [comment]
oh god the conflicts

**Comment by W3stside** [comment]
god that rebase sucked

**Comment by alfetopito** [comment]
> god that rebase sucked

Who will win the race and NOT have to do the next merge?

Competitor: https://github.com/gnosis/dex-react/pull/1111

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr1158--dexreact.review.gnosisdev.com

**Comment by alfetopito** [comment]
**Fields to have when collapsed**:

#### Orders
- Add `Status`

Also when expanded and there's the low balance warning it overlaps
screenshot_2020-06-26_08-23-18

#### Trades
- Add and rename `Amount/Received` to `Bought/Sold`
- Remove `Type`

And the `Trade` field is better as `Market` to be inline with the orders
screenshot_2020-06-26_08-25-18


Other that that, pretty good!

File: src/components/Layout/Card/Card.tsx [hunk 1]
```diff
@@ -111,6 +113,70 @@ export const CardDrawer = React.forwardRef<HTMLTableRowElement, CardDrawerProps>
   )
 })
 
+interface ResponsiveRowSizeTogglerProps {
+  handleOpen: () => void
+  openStatus: boolean
+}
+
+const ResponsiveRowWrapper = styled.td`
+  &&&&&& {
```
[the remark this answers is not in the document]
Comment (W3stside) [reply src/components/Layout/Card/Card.tsx:122]: for the `&pecificity`

**Comment by W3stside** [comment]
> At around 800px width an empty Active tab breaks layout
> src
> 
> I'm not sure I like how tab order switches from under me when Active tab becomes `order: 1`
> I would prefer same style emphasis as happens on non-mobile view. Not a strong opinion though.
> order

I had to wrap the menu as it becomes very cramped on smaller screen, consider the number of orders growing and then you'll see more of the problem - so i saw some solutions:

1. smaller font etc, looks ugly with everything else bigger AND previous "yo, this font is tew smol" complaints I figured that's not the best idea
2. column wrap, takes too much space
3. current solution, cause it's litty, i made it, i'm great, etc etc.
4. ???

**Comment by alfetopito** [comment]
> Also when expanded and there's the low balance warning it overlaps
> screenshot_2020-06-26_08-23-18

Low balance looks much better now!

What else I'd like to have:

1. Expand/collapse caret centered (now it's all the way to the right)
screenshot_2020-06-29_09-09-39

2. Add `status` to collapsed orders
3. Add `sold/bought` to collapsed trades

**Comment by W3stside** [comment]
@alfetopito 

1. The caret/chevron expander thing should stay right imo as its mobile only (most ppl right handed and it's ergonomic) but I guess I'm oppressing left handers. Shall we make a left handed mode toggle?? Hehe

**Comment by alfetopito** [comment]
>     1. The caret/chevron expander thing should stay right imo as its mobile only (most ppl right handed and it's ergonomic) but I guess I'm oppressing left handers. Shall we make a left handed mode toggle?? Hehe

Ah I see. Looks weird to me but I get it. What about making the whole row sorta click-y button-y?

**Comment by W3stside** [comment]
Good catch on ipad, fixing

> I still think the compact version of the order is not very usable. I makes me have to click on every order to see if it traded or not, for example.

... can i get an example of what u want to see then? hard to tell if i dont get feedback!

**Comment by W3stside** [comment]
@anxolin good catches on both those bugs actually, wow. Fixed, let me know what you'd like to see the collapsed cards as pls

**Comment by anxolin** [comment]
We can see it tomorrow. I'd love to define it together with @rafanator , this is why I didn't make a proposal. We can brainstorm it together

**Comment by anxolin** [comment]
There's a conversation going on with some proposals, let's close it with the outcome of it: https://gnosisinc.slack.com/archives/CSGLUC1BN/p1594030918069600

**Comment by W3stside** [comment]
## updated responsive view

### Active/Liq./Closed
Screenshot from 2020-07-07 15-51-59

### Fills/Trades
Screenshot from 2020-07-07 15-53-10

**Pull request merged**